Gem/Code/Source/Components/NetworkPlayerMovementComponent.cpp

```diff
@@ -127,8 +139,8 @@ namespace MultiplayerSample
 
         // View Axis are clamped and brought into the -1,1 range for transport across the network.
         // These are set regardless of whether or not any other player actions are allowed.
-        playerInput->m_viewYaw = MouseAxis(AZStd::clamp<float>(m_viewYaw, -cl_MaxMouseDelta, cl_MaxMouseDelta) / cl_MaxMouseDelta);
-        playerInput->m_viewPitch = MouseAxis(AZStd::clamp<float>(m_viewPitch, -cl_MaxMouseDelta, cl_MaxMouseDelta) / cl_MaxMouseDelta);
+        playerInput->m_viewYaw = MouseAxis(0.5f * AZStd::clamp<float>(m_viewYaw, -cl_MaxMouseDelta, cl_MaxMouseDelta) / cl_MaxMouseDelta);
```

Comment:
Should this send value between <-1/2, 1/2> or <-1, 1>? Looking at the comment above it seems like MouseAxis wants <-1,1>

Reply:
I think this will effectively half the mouse movement speed in mps, which should help aiming but perhaps be a but annoying when traversing around the world? We might eventually want a user-configurable scalar in the settings registry for mouse acceleration?

Reply:
You are correct! Thanks!